### PR TITLE
add pear worker

### PR DESCRIPTION
**Pull request by heyitsvi**

### CODE REVIEW

File: reference/pear/configuration.md [hunk 1]
```diff
@@ -120,6 +120,14 @@ Enable transparency. Must be set for opacity to work.
 
 Background color (Hex, RGB, RGBA, HSL, HSLA, CSS color).
 
+###  `pear.links <Object>` 
+
+Mapping of names to Pear application keys.
```
Comment (subashsn) [inline reference/pear/configuration.md]: Grammar

File: reference/pear/api.md [hunk 2]
```diff
@@ -261,6 +261,24 @@ async function logPrefUpdates () {
...
+
+The spawned worker process inherits standard input, output, and error from the parent process.
+
+A bidirectional pipe is also created which enables communication between the parent and worker process.
+
+Reference counting is handled automatically to manage the sidecar lifecycle.
+
+### `const pipe = Pear.worker.run(key <String>)`
+
+Spawns a new process by using the specified key as the entry point. Resolves to a `Duplex` stream object representing a pipe.
```
Comment (subashsn) [inline reference/pear/api.md]: "by using the" -> "with the"

File: reference/pear/api.md [hunk 3]
```diff
@@ -261,6 +261,24 @@ async function logPrefUpdates () {
...
+
+### `const pipe = Pear.worker.pipe()`
+
+Used within the worker process to get the communication pipe which is also returned by the `Pear.worker.run(key <String>)` function. Resolves to a `Duplex` stream object representing a pipe.
```
Comment (subashsn) [inline reference/pear/api.md]: Grammar improve

**Review by subashsn — CHANGES_REQUESTED** [review]

File: reference/pear/configuration.md [hunk 4]
```diff
@@ -120,6 +120,14 @@ Enable transparency. Must be set for opacity to work.
 
 Background color (Hex, RGB, RGBA, HSL, HSLA, CSS color).
 
+###  `pear.links <Object>`
```
Comment (rafapaezbas) [inline reference/pear/configuration.md]: pear.links can be an object or an array of keys. All the values stored (in the array or the object) are trusted application keys.

File: reference/pear/configuration.md [hunk 5]
```diff
@@ -120,6 +120,14 @@ Enable transparency. Must be set for opacity to work.
 
 Background color (Hex, RGB, RGBA, HSL, HSLA, CSS color).
 
+###  `pear.links <Object>` 
+
+Mapping of names to Pear application keys.
+
+#### `pear.links.worker <String>`
+
+Represents the key of the Pear application that contains the worker script.
```
Comment (rafapaezbas) [inline reference/pear/configuration.md]: might be useful to add a reference to the example app https://github.com/holepunchto/pear/blob/main/examples/desktop/app.js#L27 and https://github.com/holepunchto/pear/blob/main/examples/terminal/index.js#L21, or add the minimal example could be nice too.

**Review by subashsn — APPROVED** [review]

**Review by rafapaezbas — APPROVED** [review]

**Pull request merged**